backend/src/main/java/turip/content/controller/dto/response/WeeklyPopularFavoriteContentResponse.java

```diff
@@ -0,0 +1,17 @@
+package turip.content.controller.dto.response;
+
+import turip.content.domain.Content;
+
+public record WeeklyPopularFavoriteContentResponse(
+        ContentResponse content,
+        TripDurationResponse tripDuration
+) {
+    public static WeeklyPopularFavoriteContentResponse of(Content content,
+                                                          boolean isFavorite,
+                                                          TripDurationResponse tripDuration) {
```

Comment:
c: 이렇게 수정하면 더 가독성 좋을 것 같아요!
```suggestion
public static WeeklyPopularFavoriteContentResponse of(
        Content content,
        boolean isFavorite,
        TripDurationResponse tripDuration
) {
```